include/logger.h

```diff
@@ -94,10 +93,11 @@ int LOG_getlevel();
 void LOG_setfacility(int);
 
 /******************************* Logging Macros *******************************/
-#define ABS(__x) ({                                                            \
-  typeof(__x) _x = (__x);                                                      \
-  _x < 0 ? -1 * _x : _x;                                                           \
-})
+#define ABS(__x)                                                               \
+  ({                                                                           \
+    typeof(__x) _x = (__x);                                                    \
```

Comment:
BTW, this would break or be weird if the user passes an expression that evalutes the variable `_x` as a LHS.  Such as `ABS(_x=5)`, but that's slightly typical for these sorts of macros anyway and doesn't reflect the way we'd call `ABS()` in our code.